packages/api-sdk/src/medical/client/metriport.ts

```diff
@@ -88,12 +90,18 @@ export class MetriportMedicalApi {
    * @param options.timeout - Connection timeout in milliseconds, default 20 seconds.
    */
   constructor(apiKey: string, options: Options = {}) {
```

Comment:
nit: should we change apiKey to something else? NBD to leave as is though

Reply:
Related to the comment about not exposing `mode` - I think we should keep the SDK oriented to API Key only.